.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "python.pythonPath": "/usr/local/bin/python",
+    "python.linting.pylintArgs": [
+        "--generated-members=pyspark.*",
+        "--extension-pkg-whitelist=pyspark",
+        "--ignored-modules=pyspark.sql.functions",
+    ]
```

Comment:
I added this setting for avoiding unexpected error in VS Code
https://stackoverflow.com/questions/40163106/cannot-find-col-function-in-pyspark